previews/default_app/routes/_index.tsx

```diff
@@ -0,0 +1,41 @@
+import type { MetaFunction } from '@remix-run/node';
+
+export const meta: MetaFunction = () => {
+  return [
+    { title: 'New Remix App' },
+    { name: 'description', content: 'Welcome to Remix!' },
+  ];
+};
+
+export default function Index() {
+  return (
+    <div style={{ fontFamily: 'system-ui, sans-serif', lineHeight: '1.8' }}>
+      <h1>Welcome to Remix</h1>
```

Comment:
Not sure about this line.

Reply:
from the initial template used, will update